_test/test_mpi_wrappers/test_cluster_wrapper.m

```diff
@@ -396,22 +401,32 @@ function test_set_user_env(obj)
             [cluster,~,clOb] = obj.get_cluster_wrapper_tester(3);
             path = cluster.common_env_var('MATLABPATH');
             pathes = split(path,pathsep);
-            assertTrue(ismember(necessary_path,pathes));
-            assertFalse(ismember(test_dir,pathes));
+            assertTrue(obj.path_belongs(necessary_path,pathes));
+            assertFalse(obj.path_belongs(test_dir,pathes));
 
             addpath(test_dir);
 
             cluster = cluster.add_user_path(pc);
 
             path = cluster.common_env_var('MATLABPATH');
             pathes = split(path,pathsep);
-            assertTrue(ismember(test_dir,pathes));
-            assertTrue(ismember(necessary_path,pathes));
+
+            assertTrue(obj.path_belongs(test_dir,pathes));
+            assertTrue(obj.path_belongs(necessary_path,pathes));
+
             % remove path first to avoid warinings
             clear clPath
         end
     end
     methods(Access= protected)
+        function is = path_belongs(~,folder,path)
```

Comment:
good to have comment describing arguments

Reply:
it is tests -- it is not a user code. If you are here, you are developer and looking at the code, so it is obvious what code is doing, unlike description which may be outdated or wrong.